runner.sh

```diff
@@ -1,30 +1,43 @@
 #!/bin/bash
+
 set -e
 
-sudo touch requirements.txt
 user=nsls2
-image="condaforge"
+image="nsls2-analysis-2021-1.2"
 timestamp=$(date +%Y%m%d%H%M%S)
 env_name="nsls2-analysis-2021-1.2"
-#docker build . -t conda --target build
-#docker build . -t temp-image --target export
-#docker image build . -t $user/${image}:latest \
- #                  -t $user/${image}:${timestamp}
-#docker run -it --rm -v $PWD:/build $user/${image}:${timestamp} bash -l /build/export.sh
-#docker create -ti --name temp-container temp-image bash
-#docker cp temp-container:/build .
+
+if [ -f "nsls2-analysis-2021-1.2.tar.gz" ]
+then
+        read -p "These tar.gz and yml files already exist. Would you like to replace the existing files? " REPLACE
+        if [ "$REPLACE" == 'yes' ] || [ "$REPLACE" == 'y' ]
+                then rm nsls2-analysis-2021-1.2.tar.gz;rm nsls2-analysis-2021-1.2.yml;rm nsls2-analysis-2021-1.2-sha256sum.txt;echo "Old files have been removed."
```

Comment:
Please use the earlier suggested pattern-based removal, i.e. `rm -fv ${image}.tar.gz ${image}.yml ${image}-sha256sum.txt`. That will avoid running 3 separate `rm` commands.

Reply:
Done.